layers/+lang/semweb/packages.el

```diff
@@ -0,0 +1,37 @@
+;;; packages.el --- semweb layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2016 Sylvain Benner & Contributors
+;;
+;; Author: Andreas Textor <mail@atextor.de>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(setq semweb-packages
+  '(
+    company
+    ttl-mode
+    sparql-mode
+    ))
+
+(defun semweb/init-ttl-mode ()
+  (use-package ttl-mode
+    :mode ("\\.\\(ttl\\|n3\\)\\'" . ttl-mode)
+    :defer t))
+
+(defun semweb/init-sparql-mode ()
+  (with-eval-after-load 'company
+    (use-package sparql-mode
+      :mode ("\\.\\(sparql\\|rq\\)\\'" . sparql-mode)
+      :defer t
+      :init
+      (progn
+        (company-mode-on)
```

Comment:
Please configure company like it should be done (see other layers, for example python).